eclass/linux-mod-r1.eclass

```diff
@@ -545,10 +557,21 @@ modules_post_process() {
 	(( ${#mods[@]} )) ||
 		die "${FUNCNAME[0]} was called with no installed modules under ${path}"
 
-	# TODO?: find way for sane use with dracut (its 90kernel-modules-extra
-	# parses depmod.d files directly and assumes should include its modules
-	# which can lead to unnecessarily increased size or stale modules)
-#	_modules_process_depmod.d "${mods[@]#"${path}/"}"
```

Comment:
Can drop the function entirely if it's not going to be relevant anymore (`_modules_process_depmod.d()` is defined in case haven't noticed).

Reply:
Actually wait no, it's still relevant -- it exists for a different reason.

I'd say to leave both the comment and function alone instead, may still consider using it later on.

Reply:
(technically the comment probably needs rewording, albeit if the .conf files become conditional to USE=dist-kernel they are not a solution to the dracut problem)